components/Typography/src/index.css

```diff
@@ -0,0 +1,59 @@
+/* Headings */
+.denhaag-typography-h1,
+.denhaag-typography-h2,
+.denhaag-typography-h3,
+.denhaag-typography-h4,
+.denhaag-typography-h5 {
+  font-family: var(--denhaag-typography-font-family-heading, inherit);
+  font-weight: var(--denhaag-typography-font-weight-1);
```

Comment:
@paulAcato 
No, these are **_not_** referencing:  /proprietary/Proprietary/src/typography.tokens.json

But these are referencing: /proprietary/Components/src/denhaag/typography.tokens.json

Looks like it though.

Reply:
@robbieatacato , oké. But still, `font-weight-1` doesn't seem very readable and doesn't give me an impression of what it does. 

So since this is the font-weight of the heading i would suggest: `--denhaag-typography-font-weight-heading` and as fallback something like `--denhaag-typography-font-weight-base` or `--denhaag-typography-font-weight-bold`. Which are far more readable and understandable.

Reply:
@paulAcato headings and body/paragraph text have both 2 font-weights.
So I'll name them:
 --denhaag-typography-font-weight-normal
  --denhaag-typography-font-weight-bold